os/fs/smartfs/smartfs_utils.c

```diff
@@ -167,7 +167,7 @@ struct sector_entry_queue_s {
 struct sector_recover_info_s {
 	uint16_t totalsector;
 	uint16_t isolatedsector;
-	uint16_t cleanentries;
+	uint16_t cleanedentries;
```

Comment:
can you change cleanedentries to cleanedEntries, later one is easily readable.

Reply:
@Taejun-Kwon what do you think.
I agree, cleanedEntries could enhance readability.

However, in our smartfs module, we mostly have variable names in all lowercase.

Reply:
@Taejun-Kwon what do you think?
I agree, cleanedEntries could enhance readability.

However, in our smartfs module, we mostly have variable names in all lowercase.

Reply:
Actually I like cleanedEntries, it is good option but we will keep current style to align with others...

Reply:
Resolving this for now

Reply:
Resolving this for now.